packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1663,7 +1663,15 @@ abstract class MultiChildRenderObjectWidget extends RenderObjectWidget {
   /// objects.
   MultiChildRenderObjectWidget({ Key key, this.children = const <Widget>[] })
     : assert(children != null),
-      assert(!children.any((Widget child) => child == null)), // https://github.com/dart-lang/sdk/issues/29276
+      assert(() {
+        final int index = children.indexOf(null);
+        if (index >= 0) {
+          throw AssertionError(
+            "The widget's  children must not contain any null Widgets, but found one in index $index"
```

Comment:
I think you made some slight errors updating it here (double space between `widget's` and `children`, `null values` instead of `null Widgets`). Make sure it's grammatically correct here